bin/index.js

```diff
@@ -0,0 +1,80 @@
+#! /usr/bin/env node
```

Comment:
Please remove the files in the `bin/` directory from this PR

Reply:
Addressed @ [c58b62b](https://github.com/matter-labs/zksync-cli/pull/16/commits/c58b62b126d4298f5d68581deefb729c9dad8261)